src/components/accession2/Accession2View.tsx

```diff
@@ -0,0 +1,67 @@
+import { TabContext, TabList, TabPanel } from '@mui/lab';
```

Comment:
i just renamed the previous index.tsx to Accession2View.tsx, this is not new code

Reply:
Sure